src/ResourceManager/Network/Commands.Network/help/Add-AzureRmApplicationGatewayAuthenticationCertificate.md

```diff
@@ -30,7 +30,7 @@ The **Add-AzureRmApplicationGatewayAuthenticationCertificate** cmdlet adds an au
 Specifies the name of application gateway for which this cmdlet adds an authentication certificate.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Network.Models.PSApplicationGateway
```

Comment:
Please rereun the Update-MarkdownHelpModule command with -UseFullNameType parameter to get rid of all these changes: https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

Reply:
Done.

Reply:
Yet to do.